### DISCUSSION TITLE
Build #8846 - It works!!!

### DISCUSSION THREAD

**Discussion by linalan** [transmission/transmission#4969] [General]
I've been trying builds w/ARM support (M1 MacBook Pro) on and off for quite some time. To be blunt, they have sucked, including v4.0.0 stable when compared to 3.0 on Intel. Fast forward to last night when I tried #8846, my first nightly since v4 stable. AMAZEBALLS

* connects to peers AND maintains peers (showstopper)
* downloads from peers vs starting well and going to zero (showstopper)
* failed DL count isn't going thru the roof, actually remained at zero. (showstopper)
* upon DL completion, torrent status/color changed correctly
* upon DL completion, file properly written, .torrent and .part properly cleaned up
* no crash upon close, pause, or restart

Of the above, the first three were consistent w/every build I have tried going back into last year. The others were hit or miss. 

In any case, THANK YOU to all of those who have been toiling to get this working! Again first nightly since 4.0 stable, so not sure at which build(s) since then the "showstoppers" were fixed but 🍻

edit: Ventura 13.2.1

**Comment by ckerr** [comment]
You must be new here. We only do threads about how BAD Transmission is! 😄

**Reply by GaryElshaw** [reply]
To be fair, the initial comment was how much the ARM builds sucked.

**Reply by linalan** [reply]
I did start by saying previous releases "have sucked"! 😄 And yes, in general I tend to think everyone on the Internet is quick to complain and be negative. BUT, when things are good no one says anything which is also a problem. 

Keep up the good work and thanks again!
